### PR TITLE
[Feature] #73 #78 Editor DetailView DataBinding

### LINKED ISSUE #73: [Refactor] add init() in EditorDetailView's Entity
# Structure
> Be specific and detailed so that your team members can understand. Please assign the person in charge as well.

- Current structure
   - there was no init() in EditorDetailView's Entity so I have to make all of the temp property.

- Better structure
   - after making init() Instance can be made easier.

- after making init() Instance can be made easier.

### LINKED ISSUE #78: [Feature] EditorDetailView Data Binding
# 유저는 에디터 콘텐츠를 확인할 수 있다
- 에디터 콘텐츠 data를 불러온다.

## 🤔 Completion conditions

- [ ] make ViewModel
- [ ] make UseCase getEditorDetailContent() func 
- [ ] make fetchEditorDetailContent() func

## 👀 Related issues

- Related issue1 #73

### PR DESCRIPTION
**Pull request by rriver2**
# Issue Number
🔒 Close #73
🔒 Close #78

## Changes

- Changed files
Entity (
campair/campair/Entity/Editor/Detail/EditorDetailContent.swift
campair/campair/Entity/Editor/Detail/EquipmentContent.swift
campair/campair/Entity/Editor/Detail/OpeningSection.swift
)
View (
campair/campair/View/ContentView.swift
campair/campair/View/Editor/Detail/EditorDetailView/EditorDetailContentsVersionView.swift
campair/campair/View/Editor/Detail/EditorDetailView/EditorDetailListVersionView.swift
campair/campair/View/Editor/Detail/EditorDetailView/EditorDetailView.swift
campair/campair/View/Editor/Detail/EditorRecommendEquipmentView.swift
campair/campair/View/Editor/Detail/OpeningEditorView.swift
campair/campair/View/Editor/Detail/RecommendedEquipmentCardView.swift
)
campair/campair/Json/Editor/EditorDetailContentData.json
campair/campair/Model/EditorRepository.swift
campair/campair/UseCase/EditorUseCase.swift
campair/campair/ViewModel/Editor/EditorDetailViewModel.swift

## New features

- Json and Entity changed : To parsing Image
- make EditorDetailView's viewmodel, usecase, repository, entity

## Checklist

- [x] Is the branch you are merging on correct?
- [x] Do you comply with coding conventions?
- [x] Are there any changes not related to PR?
- [x] Has my code been self-reviewed?

### CODE REVIEW

File: campair/campair/View/Editor/Detail/OpeningEditorView.swift [hunk 1]
```diff
@@ -24,7 +25,7 @@ struct OpeningEditorView: View {
                         .font(.system(.title2))
                         .fontWeight(.bold)
                         .foregroundColor(Color.white)
-                    Image("contentCard2")
+                    Image(uiImage: UIImage(data: imageSet[openingSection.cardPaintingImageName] ?? imageSet["none"]! ) ?? UIImage(systemName: "xmark")!)
```
Comment (sustainable-git) [inline campair/campair/View/Editor/Detail/OpeningEditorView.swift:28]: Sorry that i made you confused... You should not use **xmark**. Use a spinner or gray screen to notify the user of loading.

**Review by sustainable-git — APPROVED** [review]
GGOODD!

File: campair/campair/View/Editor/Detail/OpeningEditorView.swift (hunk 1 above)
Comment (rriver2) [reply campair/campair/View/Editor/Detail/OpeningEditorView.swift:28]: okay will fix it ! thank you !

Comment (rriver2) [reply campair/campair/View/Editor/Detail/OpeningEditorView.swift]: 스크린샷 2022-06-18 오후 2 05 59
I changed like this for now, later I will add loading image !

Comment (Gobans) [reply campair/campair/View/Editor/Detail/OpeningEditorView.swift]: > 스크린샷 2022-06-18 오후 2 05 59
> 
> I changed like this for now, later I will add loading image !

Maybe we should use same loading image. We get work this together.

File: campair/campair/View/Editor/Detail/EditorRecommendEquipmentView.swift [hunk 2]
```diff
@@ -9,6 +9,7 @@ import SwiftUI
 
 struct EditorRecommendEquipmentView: View {
     let equipmentContent: EquipmentContent
+    @Binding var imageSet: [String: Data]
```
Comment (Gobans) [inline campair/campair/View/Editor/Detail/EditorRecommendEquipmentView.swift:12]: You made it by using Binding 😂, aren't this not work yesterday? Suddenly it works! good

**Review by Gobans — APPROVED** [review]
Did a good job!

File: campair/campair/View/Editor/Detail/EditorRecommendEquipmentView.swift (hunk 2 above)
Comment (rriver2) [reply campair/campair/View/Editor/Detail/EditorRecommendEquipmentView.swift:12]: You helped me so I can wrote this code ! hahahaha but I think I should study asynchronous later 🥹 ...

**Pull request merged**